### DISCUSSION TITLE
B2C conditional access using SAML instead of OIDC

### DISCUSSION THREAD

**Discussion by eliottmitchelmore** [azure-ad-b2c/samples#538] [Q&A]
I can get the Conditional Access sample working fine using OIDC but when switching to SAML it always throws an exception (undefined GUID) when the ConditionalAccessEvaluation TP is invoked.

I assume this is due to a difference between OIDC and SAML claims and the ConditionalAccessEvaluation is expecting InputClaims of a different format/type/name.  

Has anybody got this working with SAML?

**Comment by jeeszacharia** [comment]
You will need a SAML app registered in B2C to get this working.
